src/components/HomePage/BottomBar.module.css

```diff
@@ -0,0 +1,75 @@
+.fotterContainer {
+  display: flex;
+  width: 100%;
+  height: 129px;
+  justify-content: center;
+  align-items: center;
+  background-color: #535c684d;
+  padding-left: 0;
+}
+
+.fotterText {
+  font-weight: 400;
+  text-align: center;
+  font-family: Roboto;
+  color: #535c68 !important;
```

Comment:
If colours are going to be used in other places in the project, please consider checking if it's possible to place them inside a theme and import them from there.